### ISSUE TITLE
index_paranoia bad limit value error

### ISSUE THREAD

**Issue by tobsn** [TylerBrock/mongo-hacker#112]
if it's set to true we see this error on a simple doc with simple find() without any query params:

```
error: {
  "$err": "Can't canonicalize query: BadValue bad limit value in query",
  "code": 17287
}
```

**Comment by TylerBrock** [comment]
Thanks for reporting.

What does the query look like? What version of MongoDB?

**Comment by tobsn** [comment]
we found the error - I'll edit the file directly - you'll see.

**Comment by TylerBrock** [comment]
Closed by 3e776b455bbf1386ca9e68e21e0a6da913bc10f1
